### DISCUSSION TITLE
Applied for discount

### DISCUSSION THREAD

**Discussion by 01000110010** [community/community#112140] [GitHub Education]
### Hi everyone,

Hi there
I applied for the educational discount. I uploaded all relevant documentation required but I have still heard nothing back. That was roughly 2 weeks ago. What should I do?

**Comment by SebastienXia** [comment]
Hi, @01000110010 
You can try to resend it.

**Reply by 01000110010** [reply]
Hi Sebastien. I did as you suggested and still nothing. When I submit my info I am automatically taken back to the page requesting more info. Seems a bit strange. To me this looks like nothing is being submitted. I have sent both my id and a letter from the college dated from this semester. Is there anything else I can do?
